bench/compare-tree-performance.js

```diff
@@ -0,0 +1,65 @@
+const { Microcosm, update } = require('../build/microcosm')
+
+let repo = new Microcosm()
+
+let advance = (x, y) => ({ x, y })
+
+let SAMPLES = [25, 50, 100, 200]
+let WRITES = 50
+
+let results = SAMPLES.map(function(SIZE) {
+  let stats = {}
+
+  repo.addDomain('pixels', {
+    getInitialState() {
+      return {}
+    },
+    rotateHue(n) {
+      return isNaN(n) ? 0 : n + 5
+    },
+    advance(state, { x, y }) {
+      return update(state, [x, y], this.rotateHue)
```

Comment:
does giving update an array of keys update all keys to the passed value? I'm guessing this is the case but I didn't find anything in the docs.